tests/end2end/features/caret.feature

```diff
@@ -320,3 +320,25 @@ Feature: Caret mode
         And the following tabs should be open:
             - data/caret.html
             - data/hello.txt (active)
+
+    # Search + caret mode
+
+    Scenario: yanking a searched line
```

Comment:
5.8 seems to work (or is it just flaky?) but 5.9 [failed indeed](https://travis-ci.org/qutebrowser/qutebrowser/jobs/344648660#L497). FWIW I'd be fine with just skipping the tests there and not showing any error.